worker/transcribee_worker/document.py

```diff
@@ -52,12 +72,15 @@ async def _get_document_state(self) -> automerge.Document:
         doc = automerge.init(EditorDocument)
         while True:
             msg = await self.conn.recv()
-            if msg[0] == SyncMessageType.CHANGE:
-                automerge.apply_changes(doc, [msg[1:]])
-            elif msg[0] == SyncMessageType.CHANGE_BACKLOG_COMPLETE:
-                break
-            elif msg[0] == SyncMessageType.FULL_DOCUMENT:
-                doc = automerge.load(msg[1:])
+            if isinstance(msg, bytes):
+                if msg[0] == SyncMessageType.CHANGE:
+                    automerge.apply_changes(doc, [msg[1:]])
+                elif msg[0] == SyncMessageType.CHANGE_BACKLOG_COMPLETE:
+                    break
+                elif msg[0] == SyncMessageType.FULL_DOCUMENT:
+                    doc = automerge.load(msg[1:])
+            else:
```

Comment:
I don't really like the ellipsis here

Reply:
what else do you want here?
Just leave the else branch?